src/card.css

```diff
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  /* align-items: center; */
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: var(--background-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin-bottom: 40px;
+  align-self: center;
+}
+
+.card__title {
+  margin: 0 0 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--primary-text-color);
+}
+
+.card__code {
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  row-gap: 16px;
+}
+
+.card__reviews {
+  align-self: flex-end;
+
+  color: var(--primary-text-color);
+}
+
+.card__price-section {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__label {
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price {
+  color: var(--primary-text-color);
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__buy {
+  padding: 12px 69px;
+  background-color: var(--accent-color);
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  color: var(--background-color);
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  text-align: center;
+  text-transform: uppercase;
+}
+
```

Comment:
it is a good practice to add some transitions on buttons, for example => transition: color 0.5s ease;